### PR TITLE
Added modal JS to webpack code

### PR DESCRIPTION
**Pull request by autumnfound**
Signed-off-by: Martin Lowe <martin.lowe@eclipse-foundation.org>

### CODE REVIEW

**Comment by autumnfound** [comment]
I see this as a phase 1 of modal additions to the solstice assets. For phase 2, I could see us adding a mustache template as an optional render of the modal interior. This will give us more flexibility around data that is written to the page. 

We could load attributes in via a `data()` call on the related target, and adding the body to the returned object as we do currently. This object would be passed into the mustache engine and render the modal content on the fly. This would allow us to reuse the singular modal container for any pop out across the site.

File: js/solstice.modal.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * Contributors:
+ *   Martin Lowe <martin.lowe@eclipse-foundation.org>
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ */
+(function($, document) {
+  $('#eclipsefdn-modal').on('show.bs.modal', function (event) {
```
Comment (chrisguindon) [inline js/solstice.modal.js:14]: I think eclipsefdn-modal is too generic as this if for a session abstract modal.

How about we name this eclipsefdn-modal-event-session?

File: js/solstice.modal.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      var $init = $(event.relatedTarget);
+      var $modal = $(this);
+      $modal.find('h4.modal-title').text($init.attr('data-title'));
+      $modal.find('#modal-presenter').text($init.attr('data-presenter'));
```
Comment (chrisguindon) [inline js/solstice.modal.js:18]: Why are we using id and sometimes it's a class?

Comment (autumnfound) [reply js/solstice.modal.js:18]: The ones that are classes are classic modal elements and have class structure to them. The ID ones are custom attributes that were added in. I can switch it to use classes instead if you like.

**Pull request merged**